node/node_test.go

```diff
@@ -1110,6 +1111,7 @@ func TestAdminEvents_MultiSmesher(t *testing.T) {
 	cfg.FileLock = filepath.Join(cfg.DataDirParent, "LOCK")
 	cfg.SMESHING.Opts.Scrypt.N = 2
 	cfg.SMESHING.Start = false
+	cfg.API.PostListener = "0.0.0.0:10094"
```

Comment:
TODO: On Windows this doesn't work for the post service and needs to be 127.0.0.1